frontend/www/js/omegaup/carousel.config.ts

```diff
@@ -91,6 +91,30 @@ const carouselConfig: {
       href: 'https://omegaup.com/course/ResolviendoProblemas2022/',
     },
   },
+  {
+    image: '/media/homepage/discord_logo.svg',
+    title: {
+      en: 'Join our coders community',
+      es: 'Unete a nuestra comunidad de coders',
+      pt: 'Junte-se à nossa comunidade de coders',
+    },
+    description: {
+      en:
+        'Join our coders community and help us solve problems. Join our Discord server and get help from other coders.',
+      es:
+        'Únete a nuestra comunidad de coders y ayúdanos a resolver problemas. Únete al servidor de Discord y obtenga ayuda de otros coders.',
```

Comment:
```suggestion
        'Únete a nuestra comunidad de coders y ayúdanos a resolver problemas. Únete al servidor de Discord y obtén ayuda de otros coders.',
```

así se usa tú consistentemente (en vez de tú vs. usted)

Reply:
> @tvanessa @heduenas qué opinan de los textos?

Qué tal este mejor?

Únete al servidor de Discord de omegaUp y convive con la comunidad, obtén ayuda y entérate de los nuevos proyectos.
Join omegaUp´s Discord server and hang out with your community, get help and learn about new projects.